zabbixsender/zabbixsvc/zabbixsvc.go

```diff
@@ -38,25 +42,48 @@ type ZabbixResponse struct {
 
 //JSONHandler handles alerts
 type JSONHandler struct {
-	Sender      *zabbixsnd.Sender
-	KeyPrefix   string
-	DefaultHost string
+	Sender       *zabbixsnd.Sender
+	KeyPrefix    string
+	DefaultHost  string
+	DefaultHosts Hosts
```

Comment:
This is not  `DefaultHosts` this should be `Hosts`, as if host is not set we fallback onto `DefaultHost`